linux_os/guide/system/software/sudo/sudoers_validate_passwd/tests/sudoers_validate_passwd_fail.sh

```diff
@@ -0,0 +1,7 @@
+# platform = SUSE Linux Enterprise 15
```

Comment:
please rename this file so that it ends in .fail.sh, othervise it won't be detected by the test suite.

Reply:
done